src/renderer/components/ft-subscribe-button/ft-subscribe-button.scss

```diff
@@ -0,0 +1,146 @@
+.buttonList {
+  margin: 5px;
+  margin-block-end: 10px;
+  border-radius: 4px;
+  block-size: fit-content;
+  box-shadow: 0px 1px 2px rgb(0 0 0 / 50%);
+  display: flex;
+  flex-wrap: nowrap;
+  /* addresses odd clipping behavior when adjusting window size */
+  background-color: var(--primary-color);
+}
+
+.ftSubscribeButton {
+  position: relative;
+  text-align: start;
+}
+
+/* Ensures style here overrides style of .btn */
+.subscribeButton.btn {
+  min-inline-size: 150px;
+  white-space: initial;
+}
+
+.subscribeButton.btn, .profileDropdownToggle.btn {
+  align-self: center;
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+.dropdownOpened {
+  .subscribeButton, .profileDropdownToggle {
+    border-end-start-radius: 0;
+    border-end-end-radius: 0;
+  }
+}
+
+.profileDropdownToggle.btn {
+  border-inline-start: none !important;
+  border-start-start-radius: 0;
+  border-end-start-radius: 0;
+  display: inline-block;
+  min-inline-size: 1em;
+  padding-inline: 10px;
+  box-sizing: content-box;
+}
+
+.hasProfileDropdownToggle {
+  .subscribeButton.btn {
+    min-inline-size: 100px;
+    padding-inline: 5px;
+    border-inline-end: 2px solid var(--primary-color-active) !important;
+    border-start-end-radius: 0;
+    border-end-end-radius: 0;
+    box-sizing: content-box;
+  }
+}
+
+.hasProfileDropdownToggle > .subscribeButton.btn, .profileDropdownToggle.btn {
+  padding-block: 5px;
+  padding-inline: 6px;
+  box-shadow: none;
+  flex: auto;
+  block-size: 2em;
+}
+
+.profileDropdown {
+  background-color: var(--side-nav-color);
+  box-shadow: 0 1px 2px rgb(0 0 0 / 50%);
+  color: var(--secondary-text-color);
+  display: inline;
+  font-size: 12px;
+  max-block-size: 200px;
+  margin-block: -10px 0;
+  margin-inline: 5px 0;
+  overflow-y: scroll;
+  position: absolute;
+  text-align: center;
+  -webkit-user-select: none;
+  user-select: none;
+  z-index: 3;
+  // accounts for parent's left and right margins
+  inline-size: calc(100% - 10px);
+
+  .profileList {
+    list-style-type: none;
+    margin: 0;
+    padding-inline: 0;
+  }
+
+  .profile {
+    cursor: pointer;
+    display: flex;
+    gap: 0.5em;
+    padding-inline-start: 0.5em;
+    block-size: 50px;
+    align-items: center;
+    -webkit-transition: background 0.2s ease-out;
+    -moz-transition: background 0.2s ease-out;
+    -o-transition: background 0.2s ease-out;
+    transition: background 0.2s ease-out;
+
+    &:hover {
+      background-color: var(--side-nav-hover-color);
+      color: var(--side-nav-hover-text-color);
+      -moz-transition: background 0.2s ease-in;
+      -o-transition: background 0.2s ease-in;
+      transition: background 0.2s ease-in;
```

Comment:
Are the vendor prefixes needed here? If the answer is yes, the webkit one is only used in the first group, not the second one, so that should probably be added.

Reply:
Really great catch on the bug!! The vendor prefixes are the relic of me copying from elsewhere in the codebase; I'll remove all of these.